From: Kill Your TV <killyourtv@i2pmail.org>
Date: Tue, 23 Sep 2014 22:10:17 +0000
Subject: fix RaspberryPi FTBFS

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

--- a/Makefile
+++ b/Makefile
@@ -30,6 +30,7 @@
 CFLAGS += $(shell pkg-config --cflags $(DEPS))
 LDFLAGS += -pthread -lm
 LDFLAGS += $(shell pkg-config --libs $(DEPS))
+LDFLAGS += -lrt
 
 CFLAGS += $(shell dpkg-buildflags --get CFLAGS)
 LDFLAGS += $(shell dpkg-buildflags --get LDFLAGS)
