From: Kill Your TV <killyourtv@i2pmail.org>
Date: Mon, 29 Sep 2014 12:27:22 +0000
Subject: fix desktop file

---
 utox.desktop | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

--- a/utox.desktop
+++ b/utox.desktop
@@ -4,8 +4,8 @@
 Name=uTox
 Comment=A lightweight Tox client
 TryExec=utox
-Exec=utox
+Exec=utox %U
 Icon=utox
-Categories=InstantMessaging;AudioVideo;Network;
+Categories=InstantMessaging;Network;VideoConference;Telephony
 Terminal=false
 MimeType=x-scheme-handler/tox;
