From: Kill Your TV <killyourtv@i2pmail.org>
Date: Tue, 10 Mar 2015 12:08:19 +0000
Subject: fix ftbfs

bug added in 345464a1b
---
 tox.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/tox.c b/tox.c
index 860cfcf..ee68a0a 100644
--- a/tox.c
+++ b/tox.c
@@ -437,8 +437,6 @@ static void write_save(Tox *tox)
     memcpy(path_tmp + (path_len - 1), ".tmp", sizeof(".tmp"));
 
 
-    debug((char*)path_tmp);
-    debug("trying to fopen");
     file = fopen((char*)path_tmp, "wb");
     if(file) {
         fwrite(data, size, 1, file);
@@ -455,8 +453,6 @@ static void write_save(Tox *tox)
         } else {
             debug("Saved data\n");
         }
-    } else {
-        debug("no data saved...");
     }
 
 
